src/NuGet.Clients/PackageManagement.UI/Xamls/OptionsControl.xaml

```diff
@@ -32,8 +35,18 @@
         <RowDefinition Height="auto" />
         <RowDefinition Height="auto" />
         <RowDefinition Height="auto" />
+        <RowDefinition Height="auto" />
       </Grid.RowDefinitions>
 
+      <Grid Visibility="{Binding IsSolution, Converter={StaticResource BooleanToVisibilityConverter}}" Grid.Row="0" Name = "warningTextGrid" Margin="2,3,2,3">
+        <Grid.ColumnDefinitions>
+          <ColumnDefinition Width="Auto" />
+          <ColumnDefinition Width="*"/>
+        </Grid.ColumnDefinitions>
+        <Image Grid.Row="0" Grid.Column = "0" Name = "WarningTextImage"  Height="16" Width="16" Source="{StaticResource BitmapImage_StatusInformation}"/>
```

Comment:
if you add something to the start of the grid, then you should update the rest of the elements within the grid with the new positions as well.

In this case, elements in the grid were starting at row number 1, so anything you start in the grid should start with row number 1, and everything below that should have the row number changed.